src/pages/news/2020-05-26-tai-sao-the-chap-btc-la-the-manh.vn.md

```diff
@@ -0,0 +1,35 @@
+
+Để làm như vậy, tBTC yêu cầu tỷ lệ thế chấp quá 50% đối với người ký - vì vậy ETH họ phải đưa lên phải có 1,5 lần giá trị của Bitcoin được gửi. Bây giờ, những người ký vẫn được thưởng phí để thực hiện tốt công việc của họ. Nếu không, họ sẽ mất tiền. Đơn giản như vậy. 
+
+Điều này làm cho một hệ thống an toàn hơn. Bảo mật là rất quan trọng trong DeFi, đặc biệt là khi ra mắt một mạng mới. Trong khi các chốt phi tập trung khác sử dụng token gốc mới để thế chấp, Keep đã quyết định phản đối điều này; bởi vì tiêu chuẩn DeFi, ETH là loại thế chấp an toàn hơn. Ngay sau khi ra mắt, nhóm làm việc trên tBTC có kế hoạch thay đổi tỷ lệ tài sản thế chấp ETH / BTC từ 150% thành 135%.
+
+Cộng đồng đã xác định một số rủi ro bảo mật tiềm ẩn khác và thực hiện các biện pháp cẩn thận để giải quyết chúng. Trong mô hình bảo mật tBTC, nếu người đăng ký thông đồng và chạy trốn cùng tiền gửi Bitcoin của bạn, người dùng sẽ được trả lại bằng token TBTC có thể đổi thành BTC. Đó là mục đích của cam kết ETH của người ký – chúng sẽ bị tịch thu và thanh lý nếu người ký không thực hiện đúng công việc giám sát tiền gửi và các khoản hoàn BTC của người dùng. Bạn có thể đọc thêm về cách thanh lý hoạt động tBTC trong [thông số kỹ thuật](https://docs.keep.network/tbtc/index.pdf).
+
+Về nguy cơ biến động đột ngột của giá ETH? Nếu ETH giảm mạnh trong một khoảng thời gian ngắn và tất cả những người ký chạy trốn và phá vỡ chốt cùng một lúc, hệ thống sẽ rơi trở lại tổng hợp. Thêm thông tin có sẵn [ở đây](https://docs.keep.network/tbtc/index.pdf)
+
+
+### Thế chấp là một tính năng
+
+Các nhóm đóng góp cho tBTC nhận ra rằng có những liên kết BTC-ETH là các chốt cung cấp phi tập trung; các mô hình bảo mật này kém an toàn hơn tBTC. Họ dựa vào giả định trung thực lớn hơn 50% và việc họ không có ETH / tài sản thế chấp thêm để hỗ trợ cho việc tiền gửi không đủ giới hạn cho hành vi xấu. Các dự án này có xu hướng sử dụng một loại “tiền điện tử của bạn” hoàn toàn mới thay vì mã hóa t-ECDSA đã được thẩm định và được sử dụng bởi tBTC.
+
+tBTC là cách an toàn để sử dụng BTC trên Ethereum. Bảo mật là cốt lõi của nhiệm vụ dự án, và nó đã được thiết kế phù hợp. Yêu cầu người ký thế chấp là một tính năng ở đây, không phải là một lỗi. Họ cân bằng lợi ích của người dùng với những nhà cung cấp dịch vụ cho phép mạng hoạt động. Chính cam kết của người ký giúp đảm bảo an toàn cho người gửi tiền Tài sản trên mạng - trong khi vẫn trả công thích hợp cho công việc được thực hiện bởi người ký. Hoạt động hiệu quả phụ thuộc vào sự liên kết của các ưu đãi và trong trường hợp này, tài sản thế chấp của người ký là một trong những công cụ giúp mạng an toàn. 
+
+Tham gia [#tbtc Discord channel](https://discord.gg/wYezN7v) để biết thêm về tBTC. Và tham gia danh sách nhận thư [tại đây.](https://tbtc.network/#mailing-list)
```

Comment:
Ditto regarding Discord links. If you get bored and wanted to replace all of them in another PR it'd be very appreciated, otherwise I can open an issue

Reply:
I will fix all discord link for my post first, then open the PR for all other posts.

Reply:
Here we go , updated for all other posts : https://github.com/keep-network/tbtc-website/pull/217